i18n(fr): 📝 Tutorial Translation (#4688)
withastro/docs: src/content/docs/fr/tutorial/1-setup/2.mdx

---
type: tutorial
title: Créez votre premier projet Astro
description: |-
  Tutoriel : Construisez votre premier blog Astro —
  Créez un nouveau projet pour le tutoriel Astro et préparez-vous à coder
i18nReady: true
---
import Badge from '~/components/Badge.astro';
import Checklist from '~/components/Checklist.astro';
import Box from '~/components/tutorial/Box.astro';
import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
import PreCheck from '~/components/tutorial/PreCheck.astro';

<PreCheck>
  - Exécutez l'assistant de configuration `create astro` pour créer un nouveau projet Astro
  - Démarrez le serveur Astro en mode développement (dev)
  - Visualisez un aperçu en direct de votre site web dans votre navigateur
</PreCheck>

## Lancez l'assistant de configuration Astro

La méthode préférée pour créer un nouveau site Astro est d'utiliser notre assistant de configuration `create astro`.

> 1. Dans la ligne de commande de votre terminal, exécutez la commande suivante en utilisant votre gestionnaire de packages préféré :

Reviewer: `Gestionnaire de paquets` plutôt non ? (par contre j'ai un doute sur l'orthographe)
```suggestion
1. Dans la ligne de commande de votre terminal, exécutez la commande suivante en utilisant votre gestionnaire de paquets préféré :
```

    <PackageManagerTabs>
      <Fragment slot="npm">
      ```shell
      # créez un nouveau projet avec npm
      npm create astro@latest
      ```
      </Fragment>
      <Fragment slot="pnpm">
      ```shell
      # créez un nouveau projet avec pnpm
      pnpm create astro@latest
      ```
      </Fragment>
      <Fragment slot="yarn">
      ```shell
      # créez un nouveau projet avec yarn
      yarn create astro
      ```
      </Fragment>
    </PackageManagerTabs>

2. Confirmez avec `y` pour installer `create-astro`.

> 3. Lorsque l'invite vous demande : "Où souhaitez-vous créer votre nouveau projet ?", saisissez le nom d'un dossier pour créer un nouveau répertoire pour votre projet, par exemple `./tutorial`.

Reviewer 2: ```suggestion
3. À la question, "Where would you like to create your new project?" ("Où souhaitez-vous créer votre nouveau projet ?") saisissez le nom d'un dossier pour créer un nouveau répertoire pour votre projet, par exemple `./tutorial`.
```
Car dans le prompt c'est en anglais

    :::note

>     Un nouveau projet Astro ne peut être créé que dans un dossier complètement vide, alors choisissez un nom pour votre dossier qui n'existe pas déjà !

Reviewer: ```suggestion
    Un nouveau projet Astro ne peut être créé que dans un dossier complètement vide, alors choisissez un nom pour votre dossier qui n'existe pas encore !
```

    :::

> 4. Vous verrez une courte liste de modèles de départ parmi lesquels choisir. Utilisez les touches fléchées (haut et bas) pour naviguer jusqu'au modèle "Empty" (Vide), puis appuyez sur la touche Entrée pour valider votre choix.

Reviewer: Je ne sais pas comment traduire ça correctement. Je ne trouve pas que ma solution est mieux, mais "parmi lesquels choisir" est extrêmement brutal comme fin de phrase, on saisit pas son sens.
```suggestion
4. Vous verrez une courte liste de modèles de départ que vous pourrez choisir. Utilisez les touches fléchées (haut et bas) pour naviguer jusqu'au modèle “Empty” (Vide), puis appuyez sur la touche Entrée pour valider votre choix.
```

Sinon, pourquoi pas :
```suggestion
4. Vous verrez une courte liste de modèles de départ que vous pourrez sélectionner. Utilisez les touches fléchées (haut et bas) pour naviguer jusqu'au modèle “Empty” (Vide), puis appuyez sur la touche Entrée pour valider votre choix.
```

> 4. Vous verrez une courte liste de modèles de départ parmi lesquels choisir. Utilisez les touches fléchées (haut et bas) pour naviguer jusqu'au modèle "Empty" (Vide), puis appuyez sur la touche Entrée pour valider votre choix.

Reviewer 2: ```suggestion
4. Vous verrez une courte liste de modèles de départ parmi lesquels vous pourrez choisir. Utilisez les touches fléchées (haut et bas) pour naviguer jusqu'au modèle "Empty" (Vide), puis appuyez sur la touche Entrée pour valider votre choix.
```

Ou aussi (avec le clavier):
```suggestion
4. Vous verrez une courte liste de modèles de départ parmi lesquels vous pourrez choisir. Utilisez les touches fléchées (haut et bas) pour naviguer jusqu'au modèle "Empty" (Vide), puis appuyez sur la touche <kbd>Entrée</kbd> pour valider votre choix.
```

> 5. Lorsque l'invite vous demande : "Souhaitez-vous installer des dépendances ?", tapez `y`.

Reviewer 2: ```suggestion
5. À la question, "Would you like to install dependencies?" ("Souhaitez-vous installer des dépendances ?") tapez`y`.
```

6. Lorsque l'invite vous demande si vous prévoyez d'utiliser TypeScript, tapez `n`.

> 7. Lorsque l'invite vous demande : "Souhaitez-vous initialiser un nouveau référentiel Git ?", tapez `y`.

Reviewer: ```suggestion
7. Lorsque l'invite vous demande : "Souhaitez-vous initialiser un nouveau dépôt Git ?", tapez `y`.
```

> 7. Lorsque l'invite vous demande : "Souhaitez-vous initialiser un nouveau référentiel Git ?", tapez `y`.

Reviewer 2: ```suggestion
7. Lorsque l'invite demande, "Would you like to initialize a new git repository?" ("Souhaitez-vous initialiser un nouveau dépôt git ?",) tapez `y`.
```

Idem le prompt est en anglais

Une fois l'assistant d'installation terminé, vous n'avez plus besoin de ce terminal. Vous pouvez maintenant ouvrir VS Code pour continuer.

## Ouvrez votre projet dans VS Code

1. Ouvrez VS Code. Vous serez invité à ouvrir un dossier. Choisissez le dossier que vous avez créé lors de l'assistant de configuration.

2. Si c'est la première fois que vous ouvrez un projet Astro, vous devriez voir une notification vous demandant si vous souhaitez installer des extensions recommandées. Cliquez pour voir les extensions recommandées, et choisissez l'extension [Astro language support extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode). Cela fournira une coloration syntaxique et des complétions automatiques pour votre code Astro.

3. Assurez-vous que le terminal est visible et que vous pouvez voir l'invite de commande, comme ceci :

    ```sh
    utilisateur@machine:~/tutorial$
    ```

>     :::tip[Le raccourci clavier]

Reviewer 2: ```suggestion
    :::tip[Raccourci Clavier]
```
Subjectif

>     Pour basculer la visibilité du terminal, utilisez <kbd>Ctrl + J</kbd> (macOS : <kbd>Cmd ⌘ + J</kbd>).

Reviewer: ```suggestion
    Pour modifier la visibilité du terminal, utilisez <kbd>Ctrl + J</kbd> (macOS : <kbd>Cmd ⌘ + J</kbd>).
```

>     Pour basculer la visibilité du terminal, utilisez <kbd>Ctrl + J</kbd> (macOS : <kbd>Cmd ⌘ + J</kbd>).

Reviewer 2: ```suggestion
    Pour basculer l'affichage du terminal, utilisez <kbd>Ctrl + J</kbd> (macOS : <kbd>Cmd ⌘ + J</kbd>).
```
Subjectif

    :::

Vous pouvez maintenant utiliser le terminal intégré à cette fenêtre, plutôt que l'application Terminal de votre ordinateur, pour le reste de ce tutoriel.

## Exécutez Astro en mode dev

Pour prévisualiser vos fichiers de projet _comme un site web_ pendant que vous travaillez, vous devez exécuter Astro en mode développement (dev).

> ### Démarrez le serveur dev

Reviewer 2: ```suggestion
### Démarrez le serveur de dev
```
Subjectif
